benchopt/plotting/html/static/result.js

```diff
@@ -362,8 +390,8 @@ const getScatterChartLayout = () => {
     },
     xaxis: {
       type: getScale().xaxis,
-      title: 'Time [sec]',
-      tickformat: '.1e',
+      title: xaxisType === "time" ? "Time [sec]": xaxisType,
+      tickformat: xaxisType === "time" ? '.1e': '',
```

Comment:
I think the format should still be `.1e` for `tolerance` so only change it to integer for `iterations`?

Reply:
Didn't pay attention to that. Thanks for the catch!